lib/Model/Dao/PimcoreConfigBag.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model\Dao;
+
+use Pimcore\Db\PimcoreConfigStorage;
+
+/**
+ * @internal
+ */
+abstract class PimcoreConfigBag implements DaoInterface
+{
+    use DaoTrait;
+
+    /**
+     * @var array
+     */
+    protected $db;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure()
+    {
+        // nothing to do
+    }
+
+    /**
```

Comment:
How does saving work? 
Here you're calling `save()` on this dao class, but actually there's no: 
https://github.com/pimcore/pimcore/pull/9882/files#diff-de2fb2d6c9873638e6632e713bcbbf15099a44862159b86493ded6662d693b84R959

Reply:
![image](https://user-images.githubusercontent.com/6142086/127329941-d5c74313-f6ac-4ae2-bf20-fbbec398fe7a.png)

Reply:
![image](https://user-images.githubusercontent.com/6142086/127330004-a516af3a-6b1e-4a5f-bd5b-49b7c22b0181.png)